run-clang-format.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+import glob
+import argparse
+import subprocess
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-p", "--path", default="clang-format", help="Path to clang-format executable")
+args = parser.parse_args()
+
+extensions = ["cu", "cpp", "h", "cuh", "hpp"]
+file_paths = []
+
+for extension in extensions:
+    file_paths.extend(glob.glob('**/*.{}'.format(extension), recursive=True))
```

Comment:
Or in one line:

```
file_paths = [file_path for extension_paths in glob.glob('**/*.{}'.format(extension), recursive=True) for file_path in extension_paths]
```

It's less readable that way but maybe it runs faster.  No need to change this.

Reply:
Agree. I like the original way better given that we only have a few extensions and source files, so performance is unlikely to be an issue.